src/systemd/sd-id128.h

```diff
@@ -100,6 +100,9 @@ int sd_id128_get_invocation(sd_id128_t *ret);
                 ((x).bytes[15] & 15) >= 10 ? 'a' + ((x).bytes[15] & 15) - 10 : '0' + ((x).bytes[15] & 15), \
                 0 })
 
+#define SD_ID128_MAKE_STR(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p) \
+        #a #b #c #d #e #f #g #h #i #j #k #l #m #n #o #p
```

Comment:
The number of digits is checked, and it is easier to copy&paste (or even use some macro as I did) to generate that line from SD_ID128_MAKE, and later to check that those numbers match. Dunno, I think it's a slight win.